packages/fetchai/connections/soef/connection.py

```diff
@@ -348,6 +354,35 @@ async def register_service(self, oef_message: OefSearchMessage) -> None:
         handler = data_model_handlers[data_model_name]
         await handler(service_description)
 
+    async def _ping_handler(self, service_description: Description) -> None:
```

Comment:
I don't think we need a handler. The ping command should work in the background. I.e. if I register as an agent it starts pinging; until I deregister or take the connection offline.

Reply:
Ok we can keep it